src/Error/Error.php

```diff
@@ -47,28 +47,43 @@
     private $filePath;
 
     /**
-     * @param int    $type
-     * @param string $filePath
+     * @var \Exception|\Error|\Throwable|null
      */
-    public function __construct($type, $filePath)
+    private $source;
+
+    /**
+     * @param int                               $type
+     * @param string                            $filePath
+     * @param \Exception|\Error|\Throwable|null $source
```

Comment:
it was not updated for php7.1, as it is (sadly) kinda abandoned.
You may use a type of php now, right? `?int` is a valid type now